starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts

```diff
@@ -1,12 +1,16 @@
 import { Request, Response } from 'express';
 import { getReasonPhrase, StatusCodes } from 'http-status-codes';
-import { dataSource } from '../../../db/datasource';
+import { cacheRedisClient, redisHealthCheck } from '../../../cache/cache';
```

Comment:
> error TS2459: Module `"../../../cache/cache"` declares `cacheRedisClient` locally, but it is not exported.

> error TS2305: Module `"../../../cache/cache"` has no exported member `redisHealthCheck`

Reply:
Sorry about that, my IDE stopped renaming imports when I move files around. I'll investigate what happened with it, it was a feature I heavily relied upon and now it doesn't work :(